code/modules/jobs/job_types/enclave.dm

```diff
@@ -46,7 +46,7 @@
 	ADD_TRAIT(H, TRAIT_TECHNOPHREAK, src)
 	ADD_TRAIT(H, TRAIT_ENCLAVE_CODES, src)
 	H.grant_language(/datum/language/codespeak, TRUE, TRUE, LANGUAGE_MIND)
-
+	ADD_TRAIT(H, TRAIT_ABDUCTOR_TRAINING, src)
```

Comment:
Undocumented change.

Reply:
Looks like this is documented, but also don't do this. Make the subtype use a separate trait.